### DISCUSSION TITLE
Adjust UI Scale?

### DISCUSSION THREAD

**Discussion by cowcat-meoooo** [KillerInk/FreeDcam#232] [Ideas]
Any way to increase UI Scale or can this feature be added? I'm using 21:9 Phone and UI is tiny. Especially on main camera page(see image of Flash settings).
Xperia 10 V running Android 14 (68.1.A.2.244)

Thank you!
![Screenshot_20240913-033836](https://github.com/user-attachments/assets/8d7f42b9-6d62-4bca-a9ac-a4fe1113293f)

**Comment by KillerInk** [comment]
hm thats realy small and is not intented to be as it is^^
you can switch to classic theme. that should fix it.
the new theme use for the icons a font. so when you lowered the system font size its appearing like that
